src/Algebra/Properties/CommutativeSemigroup.agda

```diff
@@ -132,3 +132,41 @@ xy∙z≈yz∙x x y z =  trans (xy∙z≈y∙zx x y z) (sym (assoc y z x))
 
 xy∙z≈zx∙y :  ∀ x y z → (x ∙ y) ∙ z ≈ (z ∙ x) ∙ y
 xy∙z≈zx∙y x y z =  trans (xy∙z≈z∙xy x y z) (sym (assoc z x y))
+
+------------------------------------------------------------------------------
+-- commutative semigroup has Jordan identity
+xy∙xx≈x∙yxx : ∀ x y → (x ∙ y) ∙ (x ∙ x) ≈ x ∙ (y ∙ (x ∙ x))
+xy∙xx≈x∙yxx x y = assoc x y ((x ∙ x))
+
+------------------------------------------------------------------------------
+-- commutative semigroup is left semiMedial
+xx∙yz≈xy∙xz : ∀ x y z → (x ∙ x) ∙ (y ∙ z) ≈ (x ∙ y) ∙ (x ∙ z)
+xx∙yz≈xy∙xz x y z = begin
+  (x ∙ x) ∙ (y ∙ z)  ≈⟨ assoc x x (y ∙ z) ⟩
+  x ∙ (x ∙ (y ∙ z)) ≈⟨ ∙-congˡ (sym (assoc x y z)) ⟩
+  x ∙ ((x ∙ y) ∙ z) ≈⟨ ∙-congˡ (∙-congʳ (comm x y)) ⟩
+  x ∙ ((y ∙ x) ∙ z) ≈⟨ ∙-congˡ (assoc y x z) ⟩
+  x ∙ (y ∙ (x ∙ z)) ≈⟨ sym (assoc x y ((x ∙ z))) ⟩
```

Comment:
In this and several other proofs, the first `≈⟨` and the last `∎` are not aligned with the rest of the equational reasoning `≈⟨`s